DEPLOYING.md

```diff
@@ -0,0 +1,51 @@
+# Deploying and Development
+
+The VSCode Extension for PRQL relies on the `prqlc` code from the
+main PRQL repo. There are two Github Actions that wacth for updates
+in this directory to build and release the extension to the
+Microsoft Marketplace.
+
+## Deploying/Updating the Extension
+
+When there is a new version of PRQL (`prqlc` in particular):
+
+- In _package.json_, update:
+
+  - `version` to the proper version (from the main PRQL repo)
+  - `dependencies.prqlc` to the same version
+  - (optional) update other dependencies and devDependencies
+
+- Check _.github/workflows/pull-request.yaml_ and _.github/workflows/release.yml_ to ensure that the `node-version` is correct.
```

Comment:
I don't think this is necessary? We have to do it occasionally but it's not in the release checklist (what does "correct" mean here?)